client/src/components/TheNavbar.vue

```diff
@@ -319,16 +325,24 @@
 
 <script setup>
 import useUser from "@/composables/useUser";
+import useNotifications from "@/composables/useNotifications";
 
-import { computed } from "vue";
+import { computed, watchEffect } from "vue";
 import { useRouter } from "vue-router";
+import { useRoute } from "vue-router";
 
 const { authData, loggedIn, logout, hasElevatedRole } = useUser();
+const { getNotifications, refreshNotifications } = useNotifications();
 
 // TODO: Current year should actually be current season
 const currentYear = computed(() => new Date().getFullYear());
 
 const router = useRouter();
+const route = useRoute();
+
+watchEffect(async () => {
+  if (route.path && hasElevatedRole.value) await refreshNotifications();
```

Comment:
Hm.... wenn ich jetzt eine Notifaction bestätige, dann wird die Zahl nicht geupdated.
Entweder wir fragen nach so einer Aktion nochmal das Backend oder wir reduzieren die Zahl einfach immer um eins.

Reply:
Das ist der Nachteil der Methode das an den path zu koppeln.
man könnte das refresh natürlich dann in jede Aktion, die das beeinflußen könnte einfügen.